libs/coin-framework/src/operation.ts

```diff
@@ -225,3 +225,20 @@ export const isAddressPoisoningOperation = (
     operation.value.isZero()
   );
 };
+
+export const isEditableOperation = (
+  account: AccountLike,
+  operation: Operation
+): boolean => {
+  let isEthFamily = false;
+  if (account.type === "Account") {
+    isEthFamily = account.currency.family === "ethereum";
+  } else if (account.type === "TokenAccount") {
+    isEthFamily = account.token.parentCurrency.family === "ethereum";
+  }
+  return (
+    isEthFamily &&
+    operation.blockHeight === null &&
+    (operation.type === "OUT" || operation.type === "NFT_OUT")
```

Comment:
Don't think that line is really necessary, or at least too specific, I think you should probably just use the `operation.transactionRaw` parameter as a way to know if that transaction was crafter in the live

Reply:
fixed